es_ES/_data/catalog/widgets.json

```diff
@@ -0,0 +1,1582 @@
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/Dismissible-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "IgnorePointer",
+    "description": "Un widget que es invisible durante la prueba de aciertos. Cuando ignorar es verdadero, este widget (y su subárbol) es invisible para la prueba de aciertos. Todavía consume espacio durante el diseño y pinta a su hijo como de costumbre. Simplemente no puede ser el objetivo de eventos localizados, porque devuelve false desde RenderBox.hitTest.",
+    "categories": [
+    ],
+    "subcategories": [
+      "Touch interactions"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/IgnorePointer-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "AbsorbPointer",
+    "description": "Un widget que absorbe los punteros durante la prueba de impacto. Cuando la absorción es verdadera, este widget evita que su subárbol reciba eventos de puntero terminando la prueba de impacto en sí mismo. Todavía consume espacio durante el diseño y pinta a su hijo como de costumbre. Simplemente impide que sus hijos sean el objetivo de eventos localizados, porque devuelve true desde RenderBox.hitTest.",
+    "categories": [
+    ],
+    "subcategories": [
+      "Touch interactions"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/AbsorbPointer-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "Navigator",
+    "description": "Un widget que gestiona un conjunto de widgets hijo con una disciplina de pila. Muchas aplicaciones tienen un navegador cerca de la parte superior de su jerarquía de widgets para mostrar su historial lógico utilizando una sobreimpresión con las páginas visitadas más recientemente visualmente encima de las páginas más antiguas. El uso de este patrón permite al navegador realizar una transición visual de una página a otra moviendo los widgets en la sobreimpresión. Del mismo modo, el navegador puede ser utilizado para mostrar un diálogo posicionando el widget de diálogo sobre la página actual.",
+    "categories": [
+    ],
+    "subcategories": [
+      "Routing"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/Navigator-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "Theme",
+    "description": "Aplica un tema a los widgets de descendientes. Un tema describe los colores y las opciones tipográficas de una aplicación.",
+    "categories": [
+      "Styling"
+    ],
+    "subcategories": [
+    ],
+    "link": "https://docs.flutter.io/flutter/material/Theme-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "MediaQuery",
+    "description": "Establece un subárbol en el que las consultas de los medios de comunicación se resuelven a los datos dados.",
```

Comment:
Establishes a subtree in which media queries resolve to the given data.
Establece un sub-árbol en el cual las media queries se resuelven con los datos dados.